ScalarDiffusion.pyx

```diff
@@ -91,7 +91,6 @@ cdef class ScalarDiffusion:
 
         if 'qt' in PV.name_index:
             n_qt = PV.name_index['qt']
-            s_shift = PV.get_varshift(Gr,'s')
```

Comment:
It seems to me the changes in this file are also related to thetali.

Reply:
should I be adding back 's_shift = PV.get_varshift(Gr,'s')'?

Reply:
Yes, I think we can just keep the version of the current master for this file.